### PR TITLE
feat: add banner component

### LINKED ISSUE #280: add banner component
extract from Graphql config https://github.com/kamilkisiela/graphql-config/pull/821/files#diff-c3fd6bef8096146de56d09684b54c3e5f5bd274fa5d214ff5fb8d86f1599cb20R72

### PR DESCRIPTION
**Pull request by saihaj**
closes #280﻿

Reviewers: @PabloSzx @B2o5T

### CODE REVIEW

**Review by dotansimha — APPROVED** [review]

**Comment by B2o5T** [comment]
@saihaj I think the main feature of the banner component in `graphql-config` it's his `background` with a `linear gradient` and with `animation`, do you think we need to add that styles by default? And the possibility to change them via `props`?

Also I think that text must by centered by default, and default padding must be 15px 25ps instead 8px 8px.

Maybe also add possibility to close the banner? need to take a look at existing banners how this was implemented.

**Review by PabloSzx — CHANGES_REQUESTED** [review]
Banner is not being re-exported from the `src/index.tsx` and the project only ships a bundled version of `src/index.tsx`

**Comment by saihaj** [comment]
>  need to take a look at existing banners how this was implemented.

The current banner just sticks to the page there is no way to hide it.

**Comment by saihaj** [comment]
Not sure why the build is failing now :/

**Comment by saihaj** [comment]
@B2o5T @PabloSzx any idea why this is failing?

File: packages/components/src/components/Banner.styles.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import type { IBannerProps } from '../types/components';
+
+export const Wrapper = styled.div<IBannerProps>(
+  ({ color, bgColor, animation }) => [
+    tw`w-full sticky top-0 text-center py-4 px-6`,
+    css`
+      color: ${color || theme`colors.grayscale-line`};
+      background: ${bgColor ||
+      'linear-gradient(-45deg, #5f6184, #a7a8d7, #5f6184, #a7a8d7)'};
+      background-size: 400% 200%;
```
Comment (B2o5T) [inline packages/components/src/components/Banner.styles.ts:11]: you forget to specify this style, so there was no animation

**Pull request merged**